spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/LookupTable.scala

```diff
@@ -31,8 +32,11 @@ import scala.reflect.ClassTag
 @SerialVersionUID( - 4832171200145114633L)
 class LookupTable[T: ClassTag]
 (val nIndex: Int, val nOutput: Int, val paddingValue: Double = 0,
- val maxNorm: Double = Double.MaxValue,
- val normType: Double = 2.0, shouldScaleGradByFreq: Boolean = false)
+  val maxNorm: Double = Double.MaxValue,
+  val normType: Double = 2.0,
+  shouldScaleGradByFreq: Boolean = false,
+  wRegularizer: Regularizer[T] = null
```

Comment:
The constructor list is too long. Can we move the new constructor parameter out? and nochange in javadoc?